.travis.yml

```diff
@@ -46,4 +46,4 @@ before_script:
   - composer validate
 
 script:
-  - behat --format progress --strict
+  - bash bin/test.sh
```

Comment:
I'm wondering...
Is the `bin/test.sh` file executable (has its executable bit set)? I would assume so, as it is in the `bin` folder, which should contain executable files only. If this is the case, wouldn't it be preferable to not reference `bash` separately, but to just run `bin/test.sh`? It has a shebang that calls `bash`, so Travis should be able to figure it out from there (at least I assume so).